src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.hpp

```diff
@@ -193,7 +94,9 @@ struct DampedHarmonicHCompute : Tags::GaugeH<SpatialDim, Frame>,
  * \f{align*}
  * T_1 =& [1 - R_{H_\mathrm{init}}(t)] H_a^\mathrm{init}, \\
  * T_2 =& [\mu_{L1} \mathrm{log}(\sqrt{g}/N) + \mu_{L2} \mathrm{log}(1/N)] t_a,
- * \\ T_3 =& - \mu_S g_{ai} N^i / N. \f}
+ * \\
+ * T_3 =& - \mu_S g_{ai} N^i / N.
```

Comment:
If N, N^i are referring to lapse, shift, please change throughout to our conventions (alpha, beta^i). Same goes if g is the spatial metric instead of the spacetime metric

Reply:
I intentionally didn't do this since I don't feel like I need to clean up everything myself. My view is that the authors of the original code should clean up the documentation as were asked to many months ago. Is that okay?

Reply:
Yep! I forgot this was on @prayush and me...we’ll get this done separately!